src/main/java/at/hannibal2/skyhanni/features/misc/customscoreboard/ScoreboardElements.kt

```diff
@@ -0,0 +1,633 @@
+        { true },
+        "§710:40pm §b☽"
+    ),
+    LOBBY_CODE(
+        ::getLobbyDisplayPair,
+        { true },
+        "§8m77CK"
+    ),
+    POWER(
+        ::getPowerDisplayPair,
+        ::getPowerShowWhen,
+        "Power: Sighted"
+    ),
+    COOKIE(
+        ::getCookieDisplayPair,
+        ::getCookieShowWhen,
+        "§d§lCookie Buff\n §f3days, 17hours"
+    ),
+    EMPTY_LINE2(
+        ::getEmptyLineDisplayPair,
+        { true }, ""
+    ),
+    OBJECTIVE(
+        ::getObjectiveDisplayPair,
+        ::getObjectiveShowWhen,
+        "Objective:\n§eStar SkyHanni on Github"
+    ),
+    SLAYER(
+        ::getSlayerDisplayPair,
+        ::getSlayerShowWhen,
+        "Slayer Quest\n §7- §cVoidgloom Seraph III\n §7- §e12§7/§c120 §7Kills"
+    ),
+    EMPTY_LINE3(
+        ::getEmptyLineDisplayPair,
+        { true },
+        ""
+    ),
+    QUIVER(
+        ::getQuiverDisplayPair,
+        ::getQuiverShowWhen,
+        "Flint Arrow: §f1,234"
+    ),
+    POWDER(
+        ::getPowderDisplayPair,
+        ::getPowderShowWhen,
+        "§9§lPowder\n §7- §fMithril: §254,646\n §7- §fGemstone: §d51,234"
+    ),
+    EVENTS(
+        ::getEventsDisplayPair,
+        ::getEventsShowWhen,
+        "§7Wide Range of Events\n§7(too much to show all)"
+    ),
+    MAYOR(
+        ::getMayorDisplayPair,
+        ::getMayorShowWhen,
+        "§2Diana:\n §7- §eLucky!\n §7- §eMythological Ritual\n §7- §ePet XP Buff"
+    ),
+    PARTY(
+        ::getPartyDisplayPair,
+        ::getPartyShowWhen,
+        "§9§lParty (4):\n §7- §fhannibal2\n §7- §fMoulberry\n §7- §fVahvl\n §7- §fJ10a1n15"
+    ),
+    FOOTER(
+        ::getFooterDisplayPair,
+        { true },
+        "§ewww.hypixel.net"
+    ),
+    EXTRA(
+        ::getExtraDisplayPair,
+        ::getExtraShowWhen,
+        "§cUnknown lines the mod is not detecting"
+    ),
+    ;
+
+    override fun toString(): String {
+        return configLine
+    }
+
+    fun getPair(): List<ScoreboardElementType> {
+        return try {
+            displayPair.get()
+        } catch (e: NoSuchElementException) {
+            listOf("<hidden>" to HorizontalAlignment.LEFT)
+        }
+    }
+
+    fun isVisible(): Boolean {
+        if (!informationFilteringConfig.hideIrrelevantLines) return true
+        return showWhen()
+    }
+}
+
+
+private fun getTitleDisplayPair() = when (displayConfig.titleAndFooter.useHypixelTitleAnimation) {
+    true -> listOf(ScoreboardData.objectiveTitle to getTitleAndFooterAlignment())
+    false -> listOf(
+        displayConfig.titleAndFooter.customTitle.get().toString()
+            .replace("&", "§") to getTitleAndFooterAlignment()
+    )
+}
+
+private fun getProfileDisplayPair() =
+    listOf(CustomScoreboardUtils.getProfileTypeSymbol() + HypixelData.profileName.firstLetterUppercase() to HorizontalAlignment.LEFT)
+
+private fun getPurseDisplayPair(): List<ScoreboardElementType> {
+    var purse = PurseAPI.currentPurse.toString().formatNum()
```

Comment:
it seams wrong, there should be a method to make it without the string in between. I'm not sure if we have already a method for this.

Reply:
Wow, I look at our codebase and the change is wild
```suggestion
    var purse = PurseAPI.currentPurse.formatNum()
```